packages/agent/src/agents/supervisor/workflowController.ts

```
@@ -5,7 +5,20 @@ import { IAgent } from '../core/baseAgent.js';
 import { RunnableConfig } from '@langchain/core/runnables';
 import crypto from 'crypto';
 import { AgentType } from '../core/baseAgent.js';
-
+import fs from 'fs';
```

ESLint

[error] 'fs' is defined but never used.

(@typescript-eslint/no-unused-vars)